platform/node/src/node_map.cpp

```diff
@@ -1074,6 +1074,15 @@ NodeMap::NodeMap(v8::Local<v8::Object> options)
                            ->NumberValue()
                      : 1.0;
       }())
+    , mode([&] {
```

Comment:
Maybe a `"mode" option with possible values `"static"` and `"tile"`? Also, can you add a note about this change to the node CHANGELOG? api-gl will need an update.

Reply:
switched in https://github.com/mapbox/mapbox-gl-native/pull/10443/commits/b93ed14090fde76d3d4c13d2fd472bca44745ed1